### PR TITLE
Fix/dao-create

### PR DESCRIPTION
**Pull request by Siddharth9890**
# [PR Name]

## Type of PR

- [] - feature
- [X] - bugfix
- [] - chore
- [] - docs

## Description

Lorem Ipsum

## Checklist

- [] - Provided useful description for this PR
- [] - Provided the root cause of this bugfix
- [] - Tested on supported devices

### CODE REVIEW

File: apps/website/components/molecules/image-drop-field/image-drop-field.tsx [hunk 1]
```diff
@@ -39,22 +39,32 @@ export function ImageDropField<TFormSchema extends FieldValues = FieldValues>({
...
-      const isGif = event.target.result.toString().match('data:image/gif;');
-      if (withCrop && !isGif) {
-        imageCropDialog.onOpen(image);
-      } else {
-        onChange(image);
-      }
-    };
-
-    reader.readAsDataURL(file);
+    console.log(file);
```
Comment (R11manish) [inline apps/website/components/molecules/image-drop-field/image-drop-field.tsx:44]: remove console.log

**Review by R11manish — APPROVED** [review]

File: apps/website/components/molecules/image-drop-field/image-drop-field.tsx [hunk 2]
```diff
@@ -39,22 +39,31 @@ export function ImageDropField<TFormSchema extends FieldValues = FieldValues>({
...
-        imageCropDialog.onOpen(image);
-      } else {
-        onChange(image);
-      }
-    };
-
-    reader.readAsDataURL(file);
+    /* TODO: File size can be constant */
+    const imageTypes = ['image/jpeg', 'image/jpg', 'image/png', 'image/gif'];
+    if (file.size <= 5245329 && imageTypes.includes(file.type)) {
```
Comment (rafagomes) [inline apps/website/components/molecules/image-drop-field/image-drop-field.tsx:46]: This should be aligned to the backend to support this.
And the best practice on the front and backend is to check on both: content-type and extension.
Check: https://owasp.org/www-community/vulnerabilities/Unrestricted_File_Upload

File: apps/website/components/molecules/image-drop-field/image-drop-field.tsx [hunk 3]
```diff
@@ -39,22 +39,31 @@ export function ImageDropField<TFormSchema extends FieldValues = FieldValues>({
...
+      reader.onload = (event) => {
+        const image = event.target.result as string;
```
Comment (rafagomes) [inline apps/website/components/molecules/image-drop-field/image-drop-field.tsx:48]: When you are repeating properties...add it to a variable.
eventResult = event.target.result.
This avoid the function to go to the global event property to check that and adds more code redability

**Review by rafagomes — APPROVED** [review]
Just some improvents.

File: apps/website/components/templates/gate-view/gate-view.tsx [hunk 4]
```diff
@@ -647,9 +646,8 @@ export function GateViewTemplate({ gateProps }: GateViewProps) {
                     href={`/profile/${gateProps?.creator.username}`}
                   >
                     <Tooltip title={gateProps?.creator.name}>
-                      <Box component="a" sx={{ display: 'inline-block' }}>
+                      <Box sx={{ display: 'inline-block' }}>
```
Comment (R11manish) [inline apps/website/components/templates/gate-view/gate-view.tsx:649]: why u removed `component="a"` ?

**Pull request merged**